docs/streams/architecture.html

```diff
@@ -161,6 +161,12 @@ <h3 class="anchor-heading"><a id="streams_architecture_recovery" class="anchor-l
         Starting in 2.6, Kafka Streams will guarantee that a task is only ever assigned to an instance with a fully caught-up local copy of the state, if such an instance
         exists. Standby tasks will increase the likelihood that a caught-up instance exists in the case of a failure.
     </p>
+    <p>
+        You can also configure standby replicas with rack awareness. When configured, Kafka Streams will attempt to
+        create a standby task on a different "rack" than the active one, thus having a faster recovery time when the
```

Comment:
nit: Kafka Streams will attempt to [distribute] a standby task on a different "rack" than the active one,

Reply:
Done